test/unittests/commands/xmlupload/make_rdf_graph/test_make_values.py

```diff
@@ -11,49 +10,58 @@
+from dsp_tools.commands.xmlupload.models.intermediary.values import IntermediaryList
+from dsp_tools.commands.xmlupload.models.intermediary.values import IntermediaryRichtext
+from dsp_tools.commands.xmlupload.models.intermediary.values import IntermediarySimpleText
+from dsp_tools.commands.xmlupload.models.intermediary.values import IntermediaryTime
+from dsp_tools.commands.xmlupload.models.intermediary.values import IntermediaryUri
+from dsp_tools.commands.xmlupload.models.intermediary.values import IntervalFloats
+from dsp_tools.commands.xmlupload.models.lookup_models import IRILookup
 from dsp_tools.commands.xmlupload.models.lookup_models import JSONLDContext
-from dsp_tools.commands.xmlupload.models.lookup_models import Lookups
 from dsp_tools.commands.xmlupload.models.permission import Permissions
 from dsp_tools.commands.xmlupload.models.permission import PermissionValue
 from dsp_tools.models.exceptions import BaseError
-from dsp_tools.models.exceptions import InputError
-from dsp_tools.models.exceptions import PermissionNotExistsError
-from dsp_tools.models.exceptions import UserError
+from dsp_tools.utils.date_util import Calendar
+from dsp_tools.utils.date_util import Date
+from dsp_tools.utils.date_util import Era
+from dsp_tools.utils.date_util import SingleDate
 
 ONTO = Namespace("http://0.0.0.0:3333/ontology/9999/onto/v2#")
-namespaces = {"onto": ONTO, "knora-api": KNORA_API}
 
 PERMISSION_LITERAL = Literal("CR knora-admin:ProjectAdmin", datatype=XSD.string)
 RES_ONE_URI = URIRef("http://rdfh.ch/9999/res_one")
 
 
-@pytest.fixture
-def permissions_lookup() -> dict[str, Permissions]:
-    return {"open": Permissions({PermissionValue.CR: ["knora-admin:ProjectAdmin"]})}
+OPEN_PERMISSION = Permissions({PermissionValue.CR: ["knora-admin:ProjectAdmin"]})
+
+
+def onto_str(prop: str) -> str:
+    return f"http://0.0.0.0:3333/ontology/9999/onto/v2#{prop}"
 
 
 @pytest.fixture
-def lookups(permissions_lookup: dict[str, Permissions]) -> Lookups:
-    return Lookups(
+def lookups() -> IRILookup:
+    return IRILookup(
         project_iri="http://rdfh.ch/9999/project",
         id_to_iri=IriResolver({"res_one": "http://rdfh.ch/9999/res_one"}),
-        permissions=permissions_lookup,
-        listnodes={"testlist:node": "http://rdfh.ch/9999/node"},
-        namespaces=namespaces,
         jsonld_context=JSONLDContext({}),
     )
 
 
 class TestMakeOnePropGraphSuccess:
-    def test_boolean(self, lookups: Lookups) -> None:
+    def test_boolean(self, lookups: IRILookup) -> None:
         res_bn = BNode()
-        xml_prop = etree.fromstring("""
-        <boolean-prop name=":isTrueOrFalse">
-            <boolean permissions="open">true</boolean>
-        </boolean-prop>
-        """)
-        prop = XMLProperty.from_node(xml_prop, "boolean", "onto")
+        prop = IntermediaryBoolean(True, onto_str("isTrueOrFalse"), None, OPEN_PERMISSION)
         result, prop_name = _make_one_prop_graph(prop, res_bn, lookups)
```

Comment:
What is result, and what is prop_name? In the prod code, they are called single_prop_graph and last_prop_name. If you use the same names here, that would simplify the understanding of the tests.